bin/cheribsdtest/purecap-dynamic-mt-c18n/Makefile

```diff
@@ -0,0 +1,11 @@
+# $FreeBSD$
+
+.include <src.opts.mk>
+
+PROG=	cheribsdtest-purecap-dynamic-mt-c18n
```

Comment:
I'm not completely convinced we need `-purecap-dynamic` since the name is getting quite long and those are redundant.